### PR TITLE
Add 'see prize' when lower value

### LINKED ISSUE #5238: As a hackathon participant, 0.001 ETH on all of our prizes is inaccurate and misleading
### User Story

As a hackathon participant, 0.001 ETH on all of our prizes is inaccurate and misleading

### Problem

Previously, the 0.001 ETH display on all of our hackathon prizes was because the sponsor didn't necessary have the actual funds, the incorrect currency was being used, or Gitcoin  didn't support the correct currency at the time. As we tighten up our hackathon experience, it doesn't make sense to see all the prizes being an insignificant amount of ETH, as it subtracts from the experience.

### Description

image

### Current Behavior
prizes are all 0.001 ETH when they shouldn't be

### Expected Behavior
hackathon participants should be able to see what the prize is, straight up.

### Solutions
1. longer solution: restrict sponsors to using DAI or ETH so the issues can be posted correctly
2. stop gap solution: override amount numbers without syncing from the blockchain
3. stop gap solution: administrative controls on what to sync to the blockchain

### Definition of Done
TBD

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description

Add 'see prize' when low value
##### Refers/Fixes
Fix #5238

##### Testing

image

### CODE REVIEW

File: app/retail/templates/shared/result.html [hunk 1]
```diff
@@ -65,28 +65,32 @@
...
-        [[if tokens ]]
-          <button class="btn-expand-token" onclick="toggleExpandableBounty(event, '.[[:network]].i[[:standard_bounties_id]]')">
-            {% trans 'See All' %}   <i class="fa fa-angle-down arrow"></i>
-          </button>
-        [[/if]]
-      </div>
-      [[if value_in_usdt]]
-        <div class="tag usd">
+      [[if rounded_amount <= 0.001]]
+      <div class="tag token"><span>See Prize</span></div>
```
Comment (thelostone-mc) [inline app/retail/templates/shared/result.html]: indent this by 1?

**Comment by kuhnchris** [comment]
Uhhh hey, quick shot here, can we please also check if that's a hackathon bounty? Else this is a general "usable feature" - not sure if that is really what we want?

**Comment by danlipert** [comment]
nice!

**Comment by octavioamu** [comment]
@kuhnchris is general, if the value is 0.001 = there is no value and the rules of what you get should be on the description.

**Pull request merged**